### PR TITLE
add jeff

### PR DESCRIPTION
**Pull request by worace**

### CODE REVIEW

**Comment by worace** [comment]
@tleskin does this look ok to you?

**Comment by tleskin** [comment]
:zap:  :+1: :neckbeard:

File: visits/random_file.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pizza file lul
+
+
+hi here's a thing
+
+
+more lines
+
+
+this lin has a pyto
```
Comment (worace) [inline visits/random_file.txt]: hey @worace this line has a typo..

Comment (worace) [inline visits/random_file.txt]: @worace oh thanks for catching that i pushed a fix

**Comment by tleskin** [comment]
:poodle:

**Pull request merged**